modules/questions.py

```diff
@@ -1,29 +1,49 @@
+"""Test these functionalities before PR
+- Posting next questions
+    - Vary num, status, and tag
+- Counting questions
+    - Vary status and tag
```

Comment:
have not tried wroking out how to do these tests yet.

Reply:
I updated the instructions in `How to ask Stampy for questions.md`